evap/staff/templates/staff_course_form.html

```diff
@@ -64,10 +68,18 @@
                 {% if form.instance.state == "inEvaluation" %}
                     <div class="alert alert-warning">{% trans "You are editing a course, which is already in evaluation. Please note that only the students who did not evaluate yet will see your changes." %}</div>
                 {% endif %}
+                {% if form.instance.state == "evaluated" %}
+                    <div class="alert alert-warning">{% trans "You are editing a course, for which the evaluation already finished. Please be careful to not destroy any results." %}</div>
+                {% endif %}
+                {% if form.instance.state == "reviewed" %}
+                    <div class="alert alert-warning">{% trans "You are editing a course, which is already review. Please note that only the students who did not evaluate yet will see your changes." %}</div>
```

Comment:
`reviewed` comes after `evaluated`, the course is put into that state once all comments have been reviewed. the states are otherwise the same, so we should actually show the same message for both.